build/package/generate-download-archive-manifest.sh

```diff
@@ -17,4 +17,8 @@ set -Eeou pipefail
 
 VERSION="$(git tag --list "${TOOL_NAME}/v*" --sort=taggerdate | tail -1 | cut -d "v" -f 2)"
 
-go run ../tools/release/main.go --file "${FEED_FILE_NAME}" --version "${VERSION}"
+if [[ ${VERSION} =~ "-pre" ]]; then
+  echo "Skipping generate download center manifest for pre-release version: ${VERSION}"
+else
+  go run ../tools/release/main.go --file "${FEED_FILE_NAME}" --version "${VERSION}"
+fi
```

Comment:
given `tools/release/main.go` is go code we own why was this logic not implemented there?

Reply:
I hadn't considered it before, but now that you mention it, adding the logic here does help understand that we do not run the logic for the download centre for pre-releases. In addition, there is no need to run a go process if we already know that is not needed. Do you have a strong opinion on moving the logic there? if yes, could you clarify why? Thanks for the feedback!

Reply:
I do feel a bit strong about it since bash is not the best place to implement logic, pre releases is more than `-pre`, RCs and beta/alpha are more common
Same as goreleaser does things based on if it detects is a a [prerelease](https://goreleaser.com/customization/release/#github) I think we are ok to have the logic be internal and testable